varcode/variant_collection.py

```diff
@@ -161,10 +124,10 @@ def effects_to_string(self, *args, **kwargs):
         Arguments are passed on to variant_effects(*args, **kwargs).
         """
         lines = []
-        for variant_effect in self.variant_effects(*args, **kwargs):
+        for annotation in self.annotations(*args, **kwargs):
```

Comment:
Also I'd advocate for removing the `print_effects` method. I can print this string if I need to.